core/store/src/flat/manager.rs

```diff
@@ -82,15 +82,20 @@ impl FlatStorageManager {
     /// the shard's flat storage state hasn't been set before, otherwise it panics.
     /// TODO (#7327): this behavior may change when we implement support for state sync
     /// and resharding.
-    pub fn add_flat_storage_for_shard(&self, shard_uid: ShardUId, flat_storage: FlatStorage) {
+    pub fn add_flat_storage_for_shard(&self, shard_uid: ShardUId) {
```

Comment:
As we are here - can we fix the comment from like "Add a flat storage state for shard `shard_id`..." to "Creates `FlatStorage` for shard `shard_uid`..."?